src/main/java/org/openpnp/machine/reference/feeder/Bank.java

```diff
@@ -0,0 +1,271 @@
+/*
+
+Copyright (C) 2016 AV (CS) <phone.cri@gmail.com>
```

Comment:
The Licence is BSD 3 Clause type and compatible with GPL
and already used in this project.